int32 vehicleServoState
float32 vehicle_x_vel                            # mm/s
float32 vehicle_y_vel                            # mm/s
float32 vehicle_z_vel                            # rad/s

float32[] wheelVel                              # rad/s
float32[] servoPos                              # rad

int8[] sonar            